src/main/views/courts/edit-court-general.njk

```diff
@@ -61,6 +61,10 @@
             {% set courtTypesHtml %}
                 {% include "./tabs/types.njk" %}
             {% endset -%}
+            {% set postcodesHtml %}
+                {% include "./tabs/postcodes.njk" %}
+            {% endset -%}
+             <input id='slug' type='hidden' value='{{slug}}'></input
```

Comment:
is this hidden input needed?

Reply:
It appears to be duplicated in the diff.

Reply:
I'm not too sure about this. I know it has an effect across the board, so tempted to keep it in for now. I don't think I have a hidden one on the postcodes njk however! Correct me if I am wrong ofc

Reply:
Can you check your local copy - it looks like it's duplicated on this diff. Also, it's missing the last angle bracket in this duplicate.

Reply:
Line 67 and 82.